dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -739,4 +780,29 @@ public TopicSelection getTopicSelection() {
     public void setTopicSelection(TopicSelection topicSelection) {
         this.topicSelection = topicSelection;
     }
+
+    public Set<Author> getAuthors() {
+        T realDefaultVersion = this.getActualDefaultVersion();
+        if (realDefaultVersion != null) {
+            return realDefaultVersion.getAuthors();
+        }
+        return Set.of();
+    }
+
+    public void setAuthors(Set<Author> authors) {
+        this.authors.clear();
+        this.authors.addAll(authors);
```

Comment:
Where does this field (and the `orcidAuthors` field) get used?

Reply:
Charles will be using it in this [ticket](https://ucsc-cgl.atlassian.net/browse/DOCK-2358) to set the authors for the Zenodo DOI. The UI will also use it (not in the search page, but in these pages that list entries like https://qa.dockstore.org/workflows)